.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - iojs-v2.0.0
-  - iojs
+  - "4.0"
+  - "4.1"
```

Comment:
just `node_js: latest` will be fine for me; I have no intention of supporting LTS.